### ISSUE TITLE
Consider wider graphs

### ISSUE THREAD

**Issue by kdheepak** [lusingander/serie#7]
The graph by `serie` is a little confusing compared to git in some cases, imho:

<img width="755" alt="image" src="https://github.com/user-attachments/assets/41875855-dba9-4a19-8ebc-fe74400f5fe5">

At a glance, it looks like there's a branch from `003` out and is merged back immediately.

Thoughts?

_Originally posted by @kdheepak in https://github.com/lusingander/serie/pull/5#discussion_r1694652671_

**Comment by kdheepak** [comment]
here's how `tig` and `vim-flog` manage it:

<img width="759" alt="image" src="https://github.com/user-attachments/assets/c1010712-8fe8-4fe0-a114-4929a6cf33bd">

`tig` solves this problem by making the graph more wide

**Comment by lusingander** [comment]
I totally agree. I knew it was a problem, but I didn't fix it because I couldn't quickly enough 😢

**Comment by kdheepak** [comment]
No rush whatsoever! Just wanted to document thoughts.

**Comment by bew** [comment]
Maybe it could simply use a different color?
